File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>



int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1  || height > 8);


    for (int i = 0; i < height; i++)
        // Making lines Here
    {
        for (int spaces = height - i - 1; spaces > 0; spaces--)
        {
            // Making spaces here
            printf(" ");
        }
        printf("#");
        for (int j = 0; j < i; j++)
        {
            // Making Columns Here
            printf("#");
        }
        printf("\n");
    }

}

File: sentimental-mario-less/mario.py
# TODO
from cs50 import get_int

while True:
    height = get_int("Height: ")

    if height <= 8 and height > 0:
        break


for i in range(height):

    # printing the spaces
    for h in range(height - 1 - i):
        print(" ", end="")

    # printing the hashes
    for j in range(i + 1):
        print("#", end="")

    print()